README.md

```diff
@@ -16,11 +16,109 @@ var backlight = require('rpi-backlight');
 
 backlight.powerOn();
 backlight.powerOff();
+// 1 = Screen is OFF
+// 0 = Screen is ON
+backlight.getActualPowerValue();
```

Comment:
I know this is how it works under the hood, but we could make the API could be slightly nicer here - could you change this to `backlight.isPoweredOn()` and make it return a boolean?